client/src/components/Landing/Card.js

```diff
@@ -3,7 +3,14 @@ import {CardImg, CardBody, CardTitle, CardText} from 'reactstrap'
 import {Link} from 'react-router-dom'
 import './Card.css'
 
-export default ({title, text, imgSrc, to}) => (
+type Props = {|
+  title: string,
+  text: string,
+  imgSrc: string,
+  to: string,
+|}
+
+export default ({title, text, imgSrc, to}: Props) => (
```

Comment:
One more small thing. Exporting components directly as defaults makes them pretty hard to debug, since they name isn't shown in React dev tools. Please, define that component as a `const` with a name (probably `Card`) and then `export default Card`.